src/libraries/Microsoft.XmlSerializer.Generator/tests/SGenTests.cs

```diff
@@ -15,13 +15,19 @@ public static class SgenTests
         [Fact]
         public static void SgenCommandTest()
         {
-            string codefile = "Microsoft.XmlSerializer.Generator.Tests.XmlSerializers.cs";
+            const string CodeFile = "Microsoft.XmlSerializer.Generator.Tests.XmlSerializers.cs";
```

Comment:
> Don't we need to explicitly copy the build-time generated file?

It was generated directly into the app local directory.

> Also we should use a different name for the build-time-generated file and the test time, so that we don't mix expected/actual on rerun.

Agreed.